### DISCUSSION TITLE
Earthpy.appeears Module not Found

### DISCUSSION THREAD

**Discussion by NobodyLivesHere** [earthlab-education/Earth-Analytics-2023-01-Intro#486] [! GitHub help] [answered]
I keep getting this error when trying to create / import my analysis inside my portfolio codespace as if the module library is not accessible when trying to import.  I am led to believe that it would be an issue with the devontainer.json not syncing up properly but there seems to be no issue with that file as far as I am aware.  Do we need to copy it over for each individual assignment or is just the initial copy enough to have in the root repository folder?  I have been getting around it by just doing my own analysis in the classroom codespace and then copying the Jupyter notebook file into the portfolio repository, but it seems now I am running into errors when doing that because of the aforementioned issue with it not detecting the earthpy module. 

Edit: Double checked the .devcontainer.json in a couple other people's repositories and it seems that there was a slight discrepancy in what I had copied over (the environmental pointer was slightly different).  I changed it and committed, but the codespace itself seem to not have recognized this.  May have to delete and recreate codespace.

**Comment by edchanonline** [comment, marked as answer]
I haven't updated mine yet but planning to. My understanding is that you have to update the devcontainer because of the dependencies around earthpy (and appeears is private for now). 

Previous image in use:
`registry.hub.docker.com/earthlab/earth-analytics-python-env:latest`

Image to switch to:
`registry.hub.docker.com/earthlab/earth-analytics-dev-env:latest`

**Reply by NobodyLivesHere** [reply]
Yeah this was what it was.  I changed it but I ended up having to recreate the codespace for the portfolio because it was still trying to run off the old image.  After that everything worked smoothly.
